index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>NoteBook</title>
+    <style>
+        *{
+            margin: 0;
+            padding:0;
+            box-sizing: border-box;
+        }
+
+        body{
+            min-height: 100vh;
+            background:#d3d3d3;
+            font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+            display:flex;
+            justify-content:center;
+            align-items:center;
+        }
+        ::-webkit-scrollbar{
+            display: none;
+        }
+        .paper{
+            position:relative;
+            width: 90%;
+            max-width:800px;
+            min-width:400px;
+            height:480px;
+            margin: 0 auto;
+            background:#fafafa;
+            box-shadow:0px 2px 8px rgba(0,0,0,3);
+            border-radius:10px;
+            overflow: hidden;
+        }
+        .paper::before{
+            content:"";
```

Comment:
what will happen you have to used the `content:"";`  ??

Reply:
I have kept the content empty because if I write something inside the content it will display over the web.